x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/use_table_settings.ts

```diff
@@ -7,14 +7,22 @@
 
 import { Direction, EuiBasicTableProps, Pagination, PropertySort } from '@elastic/eui';
 import { useCallback, useMemo } from 'react';
-import { ListingPageUrlState } from '../../../../../../../common/types/common';
+// import { ListingPageUrlState } from '../../../../../../../common/types/common';
```

Comment:
I'm guessing there's some issue with importing this. I think we can remove this line, but perhaps we can add a comment for `ListingPageUrlState` so we know later to use the common type.

Reply:
no, i think this is a change i left in by mistake